package.json

```diff
@@ -17,21 +17,21 @@
   ],
   "sideEffects": false,
   "scripts": {
-    "perf": "node perf/benchmark.js",
-    "profile": "node --prof --no-logfile-per-isolate --trace-deopt --trace-opt-verbose perf/profiler.js > deopt.out && node --prof-process v8.log > v8.out",
-    "prebuild": "node prebuild/prebuild.cjs",
+    "perf": "yarn node perf/benchmark.js",
+    "profile": "yarn node --prof --no-logfile-per-isolate --trace-deopt --trace-opt-verbose perf/profiler.js > deopt.out && yarn node --prof-process v8.log > v8.out",
+    "prebuild": "yarn node prebuild/prebuild.cjs",
```

Comment:
If these scripts are run by `yarn` you don't need to use `yarn node`

Reply:
Oh, thanks for the information, then I'll get back to `node`-only version